modules/packages/Sort.chpl

```diff
@@ -574,7 +596,9 @@ iter sorted(x, comparator:?rec=defaultComparator) {
   }
 }
 
-@chpldoc.nodoc
+
+// bubble sort is generally too slow to be useful in practice
+// but, it is stable and in-place
 module BubbleSort {
```

Comment:
Since there is no proper chpldoc for this module, I suggest keeping `chpldoc.nodoc` for now, following the lead of `HeapSort` below.

Reply:
Thanks for pointing this out, I removed the nodoc in error.